<launch>
    <include file="$(find rto_bringup_sim)/launch/robot.launch" />

    <node pkg="final_project" type="finalProject.py" name="final_project" output="screen">
        <param name="learnRate" value="0.85" />
        <param name="epsilon" value="0.3" />
        <param name="gamma" value="0.8" />
        <param name="startVelocity" value="[0.5,0.0,0.0]" />
        <param name="maxEpisodes" value="5" />
    </node>
</launch>